src/pages/Dashboard/FlowRunHistory-Tile.vue

```diff
@@ -90,7 +90,11 @@ export default {
       @bar-mouseover="_barMouseover"
     >
       <template v-if="canShowTooltip" slot="tooltip">
-        <TimelineTooltip :tooltip="tooltip" :loading="tooltipLoading" />
+        <TimelineTooltip
+          :tooltip="tooltip"
+          :loading="tooltipLoading"
+          :is-dashboard="$route.name === 'dashboard'"
```

Comment:
rather than passing this prop, we could just as easily check the $route from inside the TimeLineTooltip?

Reply:
The TimelineTooltip is a functional component so I thought since it was instanceless, we wouldn't have access to `this`?

Reply:
ah, I didn't catch that. That's fine then, perhaps a different name for the prop? I think it would be better to explain what the functionality of the prop is, like `show-project-name`? What do you think?

Reply:
Agree, `show-project-name` is more descriptive. Thanks!